src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -24,37 +36,265 @@ const ActorInit En_Holl_InitVars = {
     (ActorFunc)EnHoll_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_8089A5D0[] = {
+static UNK_TYPE D_8089A550[] = { 0x465055F0, 0x00000000, 0x08000800, 0xFFFFFFFF, 0xB9B055F0, 0x00000000,
+                                 0x00000800, 0xFFFFFFFF, 0xB9B0AA10, 0x00000000, 0x00000000, 0xFFFFFFFF,
+                                 0x4650AA10, 0x00000000, 0x08000000, 0xFFFFFFFF };
+
+static UNK_TYPE D_8089A590[] = { 0xD7000000,  0xFFFFFFFF, 0xFCFFFFFF, 0xFFFDF638, 0x01004008,
+                                 &D_8089A550, 0x06000204, 0x00000406, 0xDF000000, 0x00000000 };
```

Comment:
`D_8089A550` is a Vtx and `D_8089A590` is a DList. Convert them to the proper formats please c:
I knew `D_8089A590` was a dlist because it is used in this command `gSPDisplayList(gfxP++, D_8089A590);`.
Throwing `D_8089A590` to `gfxdis` shows  `gsSPVertex(0x8089A550, 4, 0)`, so `D_8089A550` is a vtx[].
You can use `vtxdis` and `gfxdis` to properly format them. Ask in Discord if you need help with those tools

Reply:
Fixed in https://github.com/zeldaret/mm/pull/270/commits/b05bef63f958002f469901d1d3aa20d9115523f4